oats/otel-basic/fixtures/server.py

```
@@ -0,0 +1,31 @@
+
+# OATs drives acceptance tests by sending HTTP requests into a running service.
+# mise is a one-shot CLI, so this tiny shim exposes `/run` and turns that
+# request into `mise run hello` so the OTEL traces/logs can be verified end to end.
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(self):
+        if self.path != "/run":
+            self.send_response(404)
+            self.end_headers()
+            return
+
+        result = run(["mise", "run", "hello"], capture_output=True, text=True)
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)